maxim/decorators/retrieval.py

```
@@ -99,9 +117,9 @@ def sync_wrapper(*args, **kwargs):
                 # First check if the logger is available
                 maxim_logger = logger or current_logger()
                 if maxim_logger is None:
-                        raise ValueError(
-                            "[MaximSDK]: no logger found. either call this function from a @trace decorated function or pass a logger"
-                        )
+                    raise ValueError(
+                        "[MaximSDK]: no logger found. either call this function from a @trace decorated function or pass a logger"
+                    )
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)